python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_selector_group_chat.py

```diff
@@ -56,6 +64,7 @@ def __init__(
         max_selector_attempts: int,
         candidate_func: Optional[CandidateFuncType],
         emit_team_events: bool,
+        streaming: bool = False,
```

Comment:
```suggestion
        model_client_streaming: bool = False,
```

To make it consistent with `AssistantAgent`.